docs/develop/api-reference/auth-tokens.md

```diff
@@ -154,6 +156,82 @@ const TopicsPermissions = {
 };
 ```
 
+## GenerateDisposableToken API
+
+Generates a new disposable Momento auth token with the specified permissions and expiry.
+
+Disposable tokens differ from the usual Momento auth token in several key ways:
+  - They cannot be generated in the console, they can only be generated programatically. 
```

Comment:
The token passed in as input to the API must be a token with Super User scope generated via the Momento console